File: Goorm/인공지능_청소기.py
import sys

input = sys.stdin.readline
T = int(input())

for t in range(T):
	x, y, n = map(int, input().split())
	if (n - (abs(x) + abs(y))) < 0 or (n - (abs(x) + abs(y))) % 2:
		print("NO")
	else :
		print("YES")

File: Goorm/인공지능_청소기.kt
import kotlin.math.*

fun main(args: Array<String>) = with(System.`in`.bufferedReader()) {
	val T = readLine().toInt()
	repeat(T) {
		val (X, Y, N) = readLine().split(" ").map {it.toInt()}
		val temp = N - (abs(X) + abs(Y))
		if (temp >= 0 && temp % 2 == 0) {
			println("YES")
		} else {
			println("NO")
		}
	}
}